x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.tsx

```diff
@@ -191,20 +190,22 @@ export function WorkspacePanelWrapper({
               </EuiFlexGroup>
             </EuiFlexItem>
           </EuiFlexGroup>
-        </EuiPageSection>
+        </EuiPageTemplate.Section>
       )}
-      <EuiPageSection
+      <EuiPageTemplate.Section
         grow={true}
         paddingSize="none"
         contentProps={{
-          className: classNames('lnsWorkspacePanelWrapper', {
-            'lnsWorkspacePanelWrapper--fullscreen': isFullscreen,
-          }),
+          className: 'lnsWorkspacePanelWrapper__content',
         }}
+        className={classNames('lnsWorkspacePanelWrapper', {
+          'lnsWorkspacePanelWrapper--fullscreen': isFullscreen,
+        })}
+        restrictWidth={false}
```

Comment:
With `restrictWidth={false}` moved to the parent `EuiPageTemplate` component, this can be removed.

Also, to prevent a white box from peaking out behind the rounded corner edges of the workspace, we should add a `color="transparent"` here as well.

```suggestion
        color="transparent"
```